Documentation/advanced-configuration.md

```diff
@@ -0,0 +1,426 @@
+want to use separately.
+
+First we will create a new `root` bucket called `ssd` in our CRUSH map.  Under
+this new bucket we will add new `host` buckets for each node that contains an
+SSD drive so data can be replicated and used separately from the default HDD
+group.
+```bash
+# Create a new tree in the CRUSH Map for SSD hosts and OSDs
+ceph osd crush add-bucket ssd root
+ceph osd crush add-bucket node1-ssd host
+ceph osd crush add-bucket node2-ssd host
+ceph osd crush add-bucket node3-ssd host
+ceph osd crush move node1-ssd root=ssd
+ceph osd crush move node2-ssd root=ssd
+ceph osd crush move node3-ssd root=ssd
+
+# Create a new rule for replication using the new tree
+ceph osd crush rule create-simple ssd ssd host firstn
+```
+
+Secondly we will move the SSD OSDs into the new `ssd` tree, under their
+respective `host` buckets:
+```bash
+ceph osd crush set osd.1 .1102 root=ssd host=node1-ssd
+ceph osd crush set osd.3 .1102 root=ssd host=node2-ssd
+ceph osd crush set osd.7 .1102 root=ssd host=node3-ssd
+```
+
+It's important to note that the `ceph osd crush set` command requires a weight
+to be specified (our example uses `.1102`).  If you'd like to change their
+weight you can do that here, otherwise be sure to specify their original weight
+seen in the `ceph osd tree` output.
+
+So let's look at our CRUSH tree again with these changes:
+```bash
+ID WEIGHT  TYPE NAME          UP/DOWN REWEIGHT PRIMARY-AFFINITY
+-8 0.22040 root ssd
+-5 0.11020     host node1-ssd
+ 1 0.11020         osd.1           up  1.00000          1.00000
+-6 0.11020     host node2-ssd
+ 4 0.11020         osd.4           up  1.00000          1.00000
+-7 0.11020     host node3-ssd
+ 7 0.11020         osd.7           up  1.00000          1.00000
+-1 7.21828 root default
+-2 0.83509     host node1
+ 0 0.55730         osd.0           up  1.00000          1.00000
+ 2 0.27779         osd.2           up  1.00000          1.00000
+-3 1.11460     host node2
+ 3 0.55730         osd.3           up  1.00000          1.00000
+ 5 0.55730         osd.5           up  1.00000          1.00000
+-4 1.11460     host node3
+ 6 0.55730         osd.6           up  1.00000          1.00000
+ 8 0.55730         osd.8           up  1.00000          1.00000
+```
+
+### Using Disk Groups With Pools
+
+Now we have a separate storage group for our SSDs, but we can't use that storage
+until we associate a pool with it.
+The default group already has a pool called `rbd` in many cases.  If you 
+[created a pool via ThirdPartyResource](pool-tpr.md), it will use the default
+storage group as well.
+
+Here's how to create new pools:
+```bash
+# SSD backed pool with 300 (total) PGs
+ceph osd pool create ssd 300 300 replicated ssd
+```
+
+Now all you need to do is create RBD images or Kubernetes `StorageClass`es that
+specify the `ssd` pool to put it to use.
+
+## Configuring Pools
+
+### Deleting A Pool
+
+It's not possible to change the number of PGs for an existing pool.  So if
```

Comment:
Although not usually recommended PGs can be set (increased only) for existing pools - http://docs.ceph.com/docs/master/rados/operations/pools/#set-pool-values